roles/adoption_osp_deploy/tasks/prepare_undercloud.yml

```diff
@@ -0,0 +1,217 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure needed logins
+  ansible.builtin.include_tasks: login_registries.yml
+  args:
+    apply:
+      delegate_to: "osp-undercloud-0"
+
```

Comment:
Almost all the following tasks are delegated. Why not use a block to improve readability?

Reply:
done, thanks!